torch/testing/_internal/common_methods_invocations.py

```diff
@@ -70,15 +70,24 @@ def is_active(self, cls_name, test_name, device_type, dtype):
 
 
 class SkipInfo(DecorateInfo):
-    """Describes which test, or type of tests, should be skipped when testing
-       an operator. Any test that matches all provided arguments will be skipped.
-       The skip will only be checked if the active_if argument is True."""
+    """Information about which tests to skip."""
```

Comment:
I would keep the first sentence of the blurb to make it clearer that SkipInfo is used for operator testing.